content/guides/inspect-apps-with-spring-boot-admin/index.md

```diff
@@ -0,0 +1,272 @@
+{{% /callout %}}
+
+As this is a brand new project, Git source code tracking must be added. First, initialize the project with `git init`, then add all the source code to version control with `git add`, and finally, create the first commit with `git commit`. Here are the commands you need:
+
+```bash
+git init # Initialize the project as a Git project
+git add --all # Add all the source code
+git commit -am "initial commit" # Create the first commit
+```
+
+The code must then be pushed to a new remote repository on GitHub. You can do this using the GitHub command line tool as follows. 
+
+```bash
+gh repo create spring-boot-admin-server --public --source=. --remote=origin --push
+```
+
+Finally, the `tanzu` CLI can tell TAP to run Spring Boot Admin server on Kubernetes using the `apps workload create` command.
+
+```bash
+tanzu apps workload create spring-boot-admin-server \                
+    --git-repo https://github.com/benwilcock/spring-boot-admin-server \   
+    --git-branch main \
+    --type web \
+    --label app.kubernetes.io/part-of=spring-boot-admin-server \
+    --label tanzu.app.live.view=false \
+    --label tanzu.app.live.view.application.name=spring-boot-admin-server \
+    --annotation autoscaling.knative.dev/minScale=1 \
+    --namespace default \
+    --env "NAMESPACE=default" \
+    --env "DOMAIN=apps.example.com" \
+    --env "JAVA_TOOL_OPTIONS=-Dmanagement.server.port=8080" \
+    --env "SPRING_PROFILES_ACTIVE=tap" \
+    --yes
+```
+
+{{%note%}}
+Notice how the `SPRING_PROFILE_ACTIVE` environment variable has been set to `tap`. This will activate the TAP profile. The `DOMAIN` and `NAMESPACE` environment variables relate to the other customizations in the `application-tap.properties` file.
+{{%/note%}}
+
+{{%info%}}
+The Tanzu Application Platform checks out the source code, builds it into a container, and runs that container on Kubernetes. Notice how there's absolutely **no YAML** using this method!
+{{%/info%}}
+
+You can watch the build and deploy process using the `apps workload tail` command.
+
+```bash
+tanzu apps workload tail spring-boot-admin-server --since 10m --timestamp
+```
+
+## Test Your Spring Boot Admin Server on Kubernetes
+
+When the application has finished deploying, you can obtain its URL using the `apps workload get` command.
+
+```bash
+tanzu apps workload get spring-boot-admin-server
+```
+
+![Image of the tanzu cli showing the Spring Boot Admin application's URL](images/SbaUrl.png "Image of the tanzu cli showing the Spring Boot Admin application's URL")
+
+Open the URL in your browser.
+
+![Image of Spring Boot Admin running on TAP](images/SbaOnTapClip.gif "Image showing a Spring Boot Admin running on TAP.")
+
+{{%warning%}}
+If you're not using Tanzu Application Platform, expose the application as a Kubernetes service in the usual way and then navigate your browser to the application's URL.
```

Comment:
Do we want to provide a link to a resource that describes how to do this for reference? Even just a link to the [kubectl expose](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#expose) docs. We also have a piece on [Service Routing](https://tanzu.vmware.com/developer/guides/service-routing/) but perhaps that's a but overwhelming? What do you think?

Reply:
Maybe go with a link to the first one - kubectl expose. That works for me. Better simple docs than a whole guide I think.